src/Scope.ts

```diff
@@ -914,6 +925,8 @@ export class Scope {
             dispose();
         }
         this.linkSymbolTableDisposables = [];
+
+        this.cache.delete('namespaceLookup');
```

Comment:
The `namespaceLookup` was stored in the cache, and not needed outside of when the scope was linked... deleting it here makes sense